wikia.php

```diff
@@ -63,6 +63,14 @@
 		] );
 	}
 
+	// PLATFORM-1633: decrease the noise in reported transactions
+	$ex = $response->getException();
+
+	if ( $ex instanceof ControllerNotFoundException|| $ex instanceof MethodNotFoundException ) {
```

Comment:
```
wladek@wladek-pc:~$ php -r 'echo 1 instanceof Exception;'
PHP Fatal error:  instanceof expects an object instance, constant given in Command line code on line 1
```
It may also be good to check if `$ex` is an object before doing these checks.